environment_unix.yml

```diff
@@ -2,16 +2,17 @@ name: lp
 channels:
   - conda-forge
 dependencies:
-  - bison=3.4
-  - cmake
-  - flake8
-  - llvmdev=11.0.1
-  - make
-  - numpy
+  - git
   - pip
-  - python=3.10.2
+  - make
   - re2c
-  - setuptools
   - toml
   - zlib
-  - git
+  - cmake
+  - numpy
+  - flake8
+  - setuptools
+  - bison=3.4
```

Comment:
Is it needed to change the order of specifying dependencies here? It seems hard to interpret which packages where removed and which ones were added.